### DISCUSSION TITLE
part-ii/2016-75/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#152] [Question comments]
Part II 2016 Graph Theory: Paper 1, Section II, 16G 

https://questions.tripos.org/part-ii/2016-75/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

(a) This is bookwork.

(b) We use the result, proved in lectures, that a planar graph with girth $g$ has 

$$e(G) \leq \max\{\frac{g}{g-2}(n - 2), n-1\}$$

The $n-1$ case occurs when $G$ is acyclic; this should in general be checked separately but is trivial in this question because a tree is always $2$-colourable.

(i) Setting $g \leq 4$ in the formula (which corresponds to a triangle-free graph) gives $e(G) \leq 2n - 4$, so the average degree of $G$ satisfies $\frac{2e(G)}{n} \leq 4 - \frac{8}{n} < 4$. Therefore $G$ contains a vertex $x$ of degree at most $3$, and we can use the technique used to prove the six-colour theorem ($4$-colour $G-x$ by induction on $|G|$, then colour $x$ a colour not used in its neighbourhood).

(ii) Setting $g = 6$ in the formula gives $\frac{2e(G)}{n} \leq 3 - \frac{6}{n} < 3$. Therefore $G$ contains a vertex $y$ of degree at most $2$. We can $3$-colour $G - y$ by induction on $|G|$, then colour $y$ with a colour not used in its neighbourhood to get a colouring of $G$.

(iii) This is false. An odd cycle has $\chi(G) = 3$ but we can make the girth arbitrarily large by adding more vertices.

</details>
